x-pack/plugins/lists/server/services/exception_lists/utils/get_exception_list_filter.test.ts

```diff
@@ -60,7 +60,7 @@ describe('getExceptionListFilter', () => {
       savedObjectTypes: ['exception-list-agnostic', 'exception-list'],
     });
     expect(filter).toEqual(
-      '(exception-list-agnostic.attributes.list_type: list OR exception-list.attributes.list_type: list) AND exception-list-agnostic.attributes.name: "Sample Endpoint Exception List"'
```

Comment:
None of these examples demonstrate the need for the added parentheses; they all look to be logically equivalent. Is it possible to add a test for the situation causing the bug?

Reply:
The unit tests don't actually test the query logic so an extra test would be a near duplicate. I modified this one though to show the importance more clearly.